### PR TITLE
(Feature) Support terraform.workspace variable

### PR DESCRIPTION
**Pull request by Stretch96**
* Using ${terraform.workspace} currently causes `tflint` to fail:
`unknown variable accessed: terraform.workspace in terraform.tf`
* Resolves https://github.com/wata727/tflint/issues/177

### CODE REVIEW

**Comment by Stretch96** [comment]
Currently this just blindly sets `TerraformWorkspace` to `default`.

Ideally this would actually set it's value from the `terraform workspace show` output, but unsure of how to go about that. Also should decide whether it should only set it if deep check is enabled (`--deep`)

**Comment by wata727** [comment]
Thanks for your contribution!

> Ideally this would actually set it's value from the terraform workspace show output, but unsure of how to go about that. Also should decide whether it should only set it if deep check is enabled

I guess it will behave similarly to `TerraformEnv`. Since the current environment is stored in `.terraform/environment`, `TerraformWorkspace` also should refer to this.
https://github.com/wata727/tflint/blob/30023dbfbdaf3a15feb28453cefcec4c6c8d196b/config/config.go#L38-L40

**Comment by Stretch96** [comment]
Ah yes, I'll add that to the PR :+1:

**Comment by Stretch96** [comment]
@wata727 That's added to the PR now 👍

File: evaluator/evaluator.go [hunk 1]
```diff
@@ -25,6 +25,10 @@ func NewEvaluator(templates map[string]*hclast.File, schema []*schema.Template,
 		Type:  hilast.TypeString,
 		Value: c.TerraformEnv,
 	}
+	varMap["terraform.workspace"] = hilast.Variable{
```
Comment (wata727) [inline evaluator/evaluator.go]: Please add a test case for `${terraform.workspace}` syntax :)
Example:
https://github.com/wata727/tflint/blob/3cf4a7eff3d7869da768ae3e51d8699d77c4c1df/evaluator/evaluator_test.go#L267-L295

Comment (Stretch96) [reply evaluator/evaluator.go]: Yep sure, I'll add that in soon 😬

Comment (Stretch96) [reply evaluator/evaluator.go]: Ready for review :+1:

**Comment by wata727** [comment]
👍

**Pull request merged**